File: app/src/main/java/com/nhatminh/example/architecture/demoarchitecture/model/SearchResponse.java
package com.nhatminh.example.architecture.demoarchitecture.model;

import androidx.annotation.Nullable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SearchResponse {

    private Integer totalCount;

    @SerializedName("incomplete_results")
    private Boolean incompleteResults;

    @SerializedName("items")
    private List<GithubRepos> searchResults = null;

    public Integer getTotalCount() {
        return totalCount;
    }

    public Boolean getIncompleteResults() {
        return incompleteResults;
    }

    @Nullable
    public List<GithubRepos> getSearchResults() {
        return searchResults;
    }

}

File: app/src/main/java/com/nhatminh/example/architecture/demoarchitecture/model/SearchResponse.kt
package com.nhatminh.example.architecture.demoarchitecture.model

import com.google.gson.annotations.SerializedName

data class SearchResponse (

        @SerializedName("incomplete_results")
        var incompleteResults : Boolean = false,

        @SerializedName("items")
        var searchResults : MutableList<GithubRepos> = ArrayList()
)